com.spotify.EdgeClient.json

```diff
@@ -0,0 +1,276 @@
+                    "sha256": "20427b718dd130e4b66d95072c2a2bd5e17232e20dad58c1bea9da81fae330e0"
+                }
+            ]
+        },
+        {
+            "name": "spotify",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install apply_extra /app/bin",
+                "install get-scale-factor.py xsettings.py set-dark-theme-variant.py /app/bin",
+                "install spotify-bin /app/bin/spotify",
+                "install -Dm644 com.spotify.EdgeClient.appdata.xml /app/share/appdata/com.spotify.EdgeClient.appdata.xml",
+                "install -Dm644 com.spotify.EdgeClient.svg /app/share/icons/hicolor/scalable/apps/com.spotify.EdgeClient.svg",
+                "install -Dm644 com.spotify.EdgeClient-symbolic.svg /app/share/icons/hicolor/symbolic/apps/com.spotify.EdgeClient-symbolic.svg",
+                "install -Dm644 com.spotify.EdgeClient.desktop /app/share/applications/com.spotify.EdgeClient.desktop"
+            ],
+            "sources": [
+                {
+                    "type": "script",
+                    "dest-filename": "apply_extra",
+                    "commands": [
+                        "unsquashfs -quiet -no-progress spotify.snap usr/bin usr/share/spotify",
+                        "mv squashfs-root/usr/* .",
+                        "rm -r spotify.snap squashfs-root share/spotify/apt-keys share/spotify/spotify.desktop",
+                        "patchelf --replace-needed libcurl-gnutls.so.4 libcurl.so.4 bin/spotify"
+                    ]
+                },
+                {
+                    "type": "file",
+                    "path": "com.spotify.EdgeClient.desktop"
+                },
+                {
+                    "type": "file",
+                    "path": "com.spotify.EdgeClient.appdata.xml"
+                },
+                {
+                    "type": "file",
+                    "path": "com.spotify.EdgeClient.svg"
+                },
+                {
+                    "type": "file",
+                    "path": "com.spotify.EdgeClient-symbolic.svg"
+                },
+                {
+                    "type": "file",
+                    "path": "spotify-bin"
+                },
+                {
+                    "type": "file",
+                    "path": "xsettings.py"
+                },
+                {
+                    "type": "file",
+                    "path": "get-scale-factor.py"
+                },
+                {
+                    "type": "file",
+                    "path": "set-dark-theme-variant.py"
+                },
+                {
+                    "type": "extra-data",
+                    "filename": "spotify.snap",
+                    "only-arches": [
+                        "x86_64"
+                    ],
+                    "url": "https://api.snapcraft.io/api/v1/snaps/download/pOBIoZ2LrCB3rDohMxoYGnbN14EHOgD7_69.snap",
```

Comment:
what's the difference with https://github.com/flathub/com.spotify.Client/

Reply:
@hfiguiere This is the same Snap repackage but the upstream is bleeding edge and not stable. Should be realeased as a "beta" option for the same app.

Reply:
two wrong don't make a right.

Reply:
> two wrong don't make a right.

So a beta version of an app is not allowed? Shall I close my pull request?

Reply:
why not using the `beta` branch of the original package?

Reply:
It doesn't have one, so should I fork that repo and create a beta branch by myself and make a pull request?